docs/en/manuals/extensions-debugging.md

```diff
@@ -0,0 +1,151 @@
+# Debugging native code
+
+Here we describe some ways to debug an application, and also symbolicate crash logs.
+
+## Debugger
+
+When you get a particularly troublesome issue in your code,
+there are several ways to find the root cause.
+
+The most common way is to run the code via a `debugger`.
+It let's you step through the code, set `breakpoints` and it will stop the execution if you get a crash.
+
+There are several debuggers around for each platform.
+
+* Visual studio - Windows
+* VSCode - Windows, macOS, Linux
+* Android Studio - Windows, macOS, Linux
+* XCode - macOS
+* WinDBG - Windows
+* lldb / gdb - macOS, Linux, (Windows)
+* ios-deply - macOS
+
+And each tool can debug certain platforms:
+
```

Comment:
Maybe add AndroidStudio here as well, despite it being quite obvious which platform it can be used to debug?

Reply:
+1